spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/health/SpanCollectionHealthIndicator.java

```diff
@@ -0,0 +1,26 @@
+package org.springframework.cloud.sleuth.health;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.actuate.health.AbstractHealthIndicator;
+import org.springframework.boot.actuate.health.Health;
+
+/**
+ * Health indicator that shows the number of accepted / dropped spans.
+ *
+ * //TODO: should this be a health indicator (it's never down) or an Endpoint ?
```

Comment:
btw the massive circle on the left is the realtime monitored "flatMap" or dynamic merge, but in this screenshot I disabled cleaning "completed state" to accumulate a bit and replay (we will offer blackbox recorder that shows step by step the state changing within the console UI). What we don't do is of course, tracing, and the combination of both will prove massively useful for next gen of reactive microservices!